benchmark/buffers/buffer-base64-decode.js

```diff
@@ -0,0 +1,14 @@
+var assert = require('assert');
+var common = require('../common.js');
+
+var bench = common.createBenchmark(main, {});
+
+function main(conf) {
+  for (var s = 'abcd'; s.length < 32 << 20; s += s);
+  assert.equal(s.length % 4, 0);
```

Comment:
This is going to suffer performance for having not been flattened. Results for me:
```
buffers/buffer-base64-decode.js : 7.16370
```
But if you place a `s.match(/./)` (trick learned from V8 team to flatten the string internally) right after the loop, I got the following:
```
buffers/buffer-base64-decode.js : 14.13037
```
This can be likewise verified by placing `s = new Buffer(s).toString();` after the loop. Should probably flatten the string to remove testing bias.